### PR TITLE
Implement prettycron in Jobs table

### PR DESCRIPTION
**Pull request by jfurrow**

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @pierlo-upitup

File: src/js/pages/jobs/JobsTable.js [hunk 1]
```diff
@@ -105,10 +107,12 @@ class JobsTable extends React.Component {
   }
 
   renderHeadline(prop, job) {
-    const id = encodeURIComponent(job.id);
+    let {id, isGroup, name, schedules} = job;
     let itemImage = null;
+    const jobID = encodeURIComponent(id);
```
Comment (rcorral) [inline src/js/pages/jobs/JobsTable.js]: you could just do `id = encodeURIComponent(id);` so that you don't have to change the lines below.

Comment (jfurrow) [inline src/js/pages/jobs/JobsTable.js]: Done!

File: src/js/pages/jobs/JobsTable.js [hunk 2]
```diff
@@ -128,6 +133,26 @@ class JobsTable extends React.Component {
...
     }
 
+    if (schedules && schedules.length !== 0) {
+      let schedule = schedules[0];
+
+      if (schedule.enabled) {
+        scheduleIcon = (
+          <Tooltip
+            wrapperClassName="tooltip-wrapper icon icon-margin-left"
+            content={prettycron.toString(schedule.cron)}
```
Comment (kennyt) [inline src/js/pages/jobs/JobsTable.js]: this actually seems like something that would be nice on it's own column. but that's another issue

**Comment by kennyt** [comment]
LGTM

**Pull request merged**